### DISCUSSION TITLE
Reallive examples are no longer available since 1.0 release

### DISCUSSION THREAD

**Discussion by marvinhuebner** [Dave136/vue-email#199] [General]
Before the 1.0 release there were some real-live examples in the documentation. I started on this basis a few weeks ago and actually found it quite helpful. Are these templates still archived somewhere? I can't find them in the repository.

Even though the API has changed with the 1.0 release, it would be cool to have access to these ( even if outdated) templates somehow.

Thank you and best regards,
Marvin

**Comment by Flowko** [comment]
hi, yeah something im yet to get to, for older emails you can visit : https://vue-email-demo.vercel.app/

you can also just go to https://react.email/examples the code will kinda be the same, just have to use vue 3 logic instead of react
